src/event_timer.py

```diff
@@ -83,7 +83,7 @@ def ensure_event_timer(self, event_name: str, interval: int, timeout: Optional[i
         self._render_event_template("service", event_name, context)
         self._render_event_template("timer", event_name, context)
         try:
-            # Binding for systemctl do no exist, so `subprocess.run` used.
+            # Binding for systemctl do not exist, so `subprocess.run` used.
```

Comment:
I think we can use sytemd libs here.
https://github.com/canonical/operator-libs-linux/blob/main/lib/charms/operator_libs_linux/v1/systemd.py
https://charmhub.io/operator-libs-linux